src/formatting_layer.rs

```diff
@@ -75,6 +76,18 @@ impl<W: for<'a> MakeWriter<'a> + 'static> BunyanFormattingLayer<W> {
             pid: std::process::id(),
             hostname: gethostname::gethostname().to_string_lossy().into_owned(),
             bunyan_version: 0,
+            default_fields: Vec::new(),
+        }
+    }
+
+    pub fn with_default_fields(name: String, make_writer: W, default_fields: Vec<(String, Value)>) -> Self {
```

Comment:
I'd prefer to use `HashMap` here instead of `Vec` given that we don't want repeated keys.

Reply:
Fixed